src/components/title/index.ts

```diff
@@ -0,0 +1 @@
+export { EuiTitle, EuiTitleSize, EuiTitleTextTransform } from './title';
```

Comment:
These shouldn't be exported in the index

```suggestion
export { EuiTitle } from './title';
```

Reply:
Currently there is a dependency in
`src/components/empty_prompt/index.d.ts` for `EuiTitleSize`

Reply:
Since the EuiEmptyPrompt was already TS, the only place the TS def existed was in `index.d.ts` but now it can be accessed directly from `title.tsx`.

@chandlerprall Should we be exporting the props in the index file or make the dependency's import path at the individual component file level?

Reply:
It's more common in our components for these supporting values to be imported directly from component's file instead of through the index. While there hasn't been a hard rule around this, I think it should be viewed as _"things in **index** are expected to be consumed outside of EUI"_; in this case those values shouldn't be exposed.

Reply:
PR updated